@types/pal/image.d.ts

```diff
@@ -0,0 +1,65 @@
+declare module 'pal/image' {
+    export type ImageSource = import('pal/image/types').ImageSource;
+    export type IMemoryImageSource = import('pal/image/types').IMemoryImageSource;
+    export type PixelFormat = import('cocos/asset/assets/asset-enum').PixelFormat;
+    export class ImageData {
+        constructor (imageAsset?: ImageSource | ArrayBufferView);
+        /**
+         * Destroy resources.
+         */
+        destroy (): void;
+
+        /**
+         * Get and set image data source.
+         * @param value Image data source.
+         */
+        get data(): ImageSource | ArrayBufferView;
+        set data(value: ImageSource | ArrayBufferView);
+
+        /**
+         * Get image width.
+         */
+        get width(): number;
+        /**
+         * Set image width(when source is IMemoryImageSource).
+         */
+        set width(value: number);
+
+        /**
+         * Get image height.
+         */
+        get height(): number;
+        /**
+         * Set image height(when source is IMemoryImageSource).
+         */
+        set height(value: number);
+
+        /**
+         * Get image format(when source is IMemoryImageSource).
+         */
+        get format(): PixelFormat;
+        /**
+         * Get if the image is compressed.
+         */
+        get compressed(): boolean;
+        get mipmapLevelDataSize(): number[] | undefined;
+        /**
+         * Load image via local url or web url.
+         */
+        static loadImage (url: string): Promise<ImageData>
```

Comment:
```suggestion
        static loadImage (url: string, options: Record<string, unknown>): Promise<ImageData>
```
we should support load options, see this use case

https://github.com/cocos/cocos-engine/blob/2d67b2d6c7ed604c4937eefc5f697b762e205398/cocos/asset/asset-manager/download-dom-image.ts#L25-L35

Reply:
The options parameter is not implemented, and it feels unnecessary to pass in the options parameter.